### PR TITLE
infra for MDN->AWS

### PR DESCRIPTION
**Pull request by metadave**
- creates EFS, memcached and redis resources per [dev, stage, prod], with [stage, prod] commented out for now to save $$$.
- all resources are tagged with a `Stack` tag for use with AWS Resource Groups

parent: https://github.com/mozmeao/infra/issues/378

### CODE REVIEW

File: apps/mdn/mdn-aws/infra/multi_region/tf/main.tf [hunk 1]
```diff
@@ -9,3 +9,94 @@ terraform {
...
+    redis_node_size = "cache.t2.micro"
+    redis_num_nodes = 1
+    subnets = "${var.subnets}"
+    nodes_security_group = "${var.nodes_security_group}"
+}
+
+#module "redis-stage" {
+#    source = "redis"
+#    redis_name = "stage"
+#    redis_node_size = "cache.t2.micro"
```
Comment (metadave) [inline apps/mdn/mdn-aws/infra/multi_region/tf/main.tf]: per a discussion with @escattone, let's use 3 x `cache.t2.small` for staging and possibly dev

**Comment by jgmize** [comment]
LGTM so far, no objections to uncommenting the sections that I looked at or merging as-is and iterating later. Let me know if you'd like me to take another look at any additional changes to this WIP PR, otherwise I'm fine with merging, applying, and iterating with another PR.

**Comment by metadave** [comment]
woo, just messed up rebase, I'll fix this soonish.

**Comment by metadave** [comment]
closing in favor of https://github.com/mozmeao/infra/pull/395

**Comment by metadave** [comment]
going to leave this branch here for a bit to ensure I didn't miss anything when moving to a new branch

**Pull request closed without merging**